raiden-cli/src/disclaimer.json

```diff
@@ -1 +1 @@
ttps://www.youtube.com/channel/UCoUP_hnjUddEvbxmtNCcApg             │\n │ - Read the blog posts on Medium: https://medium.com/@raiden_network    │\n │                                                                        │\n │ Terms of Use                                                           │\n │ By accessing the Raiden Light Client through our website you agree to  │\n │ the Terms of Use and Privacy Policy:                                   │\n │ https://github.com/raiden-network/light-client/blob/master/TERMS.md    │\n │ https://raiden.network/privacy.html                                    │\n │                                                                        │\n │ Disclaimer                                                             │\n │ The Raiden Light Client is a beta version of experimental open source  │\n │ software released as a test version under an MIT license and may       │\n │ contain errors and/or bugs. No guarantee or representation whatsoever  │\n │ is made regarding its suitability (or its use) for any purpose or      │\n │ regarding its compliance with any applicable laws and regulations.     │\n │ Use of the Raiden Light Client is at your own risk and discretion and  │\n │ by using the software you warrant and represent that you have read     │\n │ this disclaimer, understand its contents, assume all risk related      │\n │ thereto and hereby release, waive, discharge and covenant not to hold  │\n │ us or any of our officers, employees or affiliates from and for any    │\n │ direct or indirect damage resulting from the Raiden Light Client or    │\n │ the use thereof. Such to the extent as permissible by applicable laws  │\n │ and regulations.                                                       │\n │                                                                        │\n │ Privacy warning                                                        │\n │ Please be aware, that by using the Raiden Light Client, among others   │\n │ your Ethereum address, channels, channel deposits, settlements and the │\n │ Ethereum address of your channel counterparty will be stored on the    │\n │ Ethereum chain, i.e. on servers of Ethereum node operators and ergo    │\n │ are to a certain extent publicly available. The same might also be     │\n │ stored on systems of parties running Raiden nodes connected to the     │\n │ same token network. Data present in the Ethereum chain is very unlikely│\n │ to be able to be changed, removed or deleted from the public arena.    │\n │                                                                        │\n │ Also be aware, that data on individual Raiden token transfers will be  │\n │ made available via the Matrix protocol to the recipient, intermediating│\n │ nodes of a specific transfer as well as to the Matrix server operators,│\n │ see Raiden Transport Specification.                                    │\n └────────────────────────────────────────────────────────────────────────┘\n\n"
```

Comment:
Can you remove the explicit line breaks? /n There is a risk, that the disclaimer is not readable on every screen (especially on mobile).

Reply:
That is for the CLI, if I remove them it will just be a long string of text. :)